Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -1851,6 +1855,24 @@ extension TelemetryWrapper {
                 let properties = GleanMetrics.AppErrors.LargeFileWriteExtra(size: quantity)
                 GleanMetrics.AppErrors.largeFileWrite.record(properties)
             }
+        case(.action, .tap, .fxSuggest, _, let extras ):
+            if let suggestion = extras?[EventValue.fxSuggestion.rawValue] as? RustFirefoxSuggestion {
+                GleanMetrics.FxSuggest.pingType.set("fxsuggest-click")
+                //GleanMetrics.FxSuggest.contextId.set(GleanMetrics.TopSites.contextId)
```

Comment:
I tried getting the contextId but I got a type mismatch of "UuidMetricType" and "UUID"

Reply:
Hello! Just want to understand, you need to have the same context ID as the sponsored tile context id? If so we can make some adjustment so this is available, but as is indeed you won't be able to access it from the Glean SDK (as it's not a getter method in that way).

Reply:
Ok I create [a task](https://github.com/mozilla-mobile/firefox-ios/issues/17600) so we make this ID available. Any timeline I need to be aware of for this work?

Reply:
Ok [here's the PR](https://github.com/mozilla-mobile/firefox-ios/issues/17600) to access the context id. Once merged, you'll be able to access the contextual id through `TelemetryContextualIdentifier.contextId`. Let me know if this doesn't work out!

Reply:
Thanks so much for landing that, @lmarceau—and sorry I missed your query about the timeline! We're targeting 124 for QA and rollout, but having the pieces in earlier is definitely helpful ☺️

Reply:
It didn't take long, so might as well do it now 😄 Let me know if anything else is needed on my end

Reply:
@lmarceau  tyty